package.json

```diff
@@ -611,6 +611,14 @@
         ],
         "visible": true
       },
+      {
+        "permissionName": "ui-users.loans.anonymize",
+        "displayName": "Users: User loans anonymize",
+        "subPermissions": [
+          "ui-users.view"
```

Comment:
I don't think you should inherit view permission, please read ticket scenarios and permissions `view` introduces

Reply:
My mistake, forgot to get rid of subPermissions